### DISCUSSION TITLE
My Android 7.1.2 deceive‘s IPv6 address disappeared

### DISCUSSION THREAD

**Discussion by CCCAUCHY** [termux/termux-app#3491] [Q&A] [answered]
The public IPv6 address of my android device's wlan0 network interface disappears after a while on the lock screen.
How to fix this?

**Comment by agnostic-apollo** [comment, marked as answer]
Probably wifi shuts down, in wifi preferences, there should be an option for keep wifi on when screen if off.  Closing since issue is not related to termux.

**Reply by CCCAUCHY** [reply]
wifi preferences have been set, it will never shut down and I can use ipv4 to connect it anytime.
